stdlib/jdbc/src/main/ballerina/src/java.jdbc/Module.md

```diff
@@ -3,6 +3,9 @@
 This module provides the functionality required to access and manipulate data stored in any type of relational database 
 that is accessible via Java Database Connectivity (JDBC).
 
+**Prerequisite:** Please copy the JDBC driver corresponding to the database you are trying to interact with, 
```

Comment:
This is not the case with the .jar executable. The correct way should be defining the dependancy in the config IIRC.